lib/jvm.sh

```diff
@@ -70,6 +71,7 @@ get_full_jdk_version() {
   "18") echo "${DEFAULT_JDK_18_VERSION}" ;;
   "19") echo "${DEFAULT_JDK_19_VERSION}" ;;
   "20") echo "${DEFAULT_JDK_20_VERSION}" ;;
+  "21") echo "${DEFAULT_JDK_20_VERSION}" ;;
```

Comment:
```suggestion
  "21") echo "${DEFAULT_JDK_21_VERSION}" ;;
```

Reply:
Should the tests be tweaked to catch this? Currently they didn't fail even with the typo.

Reply:
Yeah, that's super bad. The tests test that Java is installed and all the checks pass - but not if the correct version is installed. 🤔

Reply:
Tracking here: https://github.com/heroku/heroku-buildpack-jvm-common/issues/277